service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/CompositionMapper.java

```diff
@@ -78,12 +77,8 @@ public Composition mapComposition(POCDMT000002UK01ClinicalDocument1 clinicalDocu
                 .setTarget(relatedDocIdentifier);
         }
 
-        if (clinicalDocument.sizeOfAuthorArray() > 0) {
-            for (POCDMT000002UK01Author author : clinicalDocument.getAuthorArray()) {
-                Encounter.EncounterParticipantComponent authorComponent = (authorMapper.mapAuthorIntoParticipantComponent(author));
-                composition.addAuthor(authorComponent.getIndividual());
-            }
-        }
+        practitionerRoles.stream()
+            .forEach(it -> composition.addAuthor(it.getPractitioner()));
```

Comment:
Can npe be thrown here and would it be better to use .foreach(..) instead of stream().foreach(..)